tasks/src/zero-shot-classification/about.md

```diff
@@ -0,0 +1,20 @@
+# Zero Shot Classification
+
+## About the Task
+
+Zero Shot Classification ([ZSC](https://joeddav.github.io/blog/2020/05/29/ZSL.html)) is a subset of [transfer learning](https://en.wikipedia.org/wiki/Transfer_learning) and an attribute of Large Language Models (LLM/s), effectively we use a large pretrained language model to generate a sequence of text that answers an informal question in the form of a prompt. The "prompt" is basically context for some task, [prompt engineering](https://en.wikipedia.org/wiki/Prompt_engineering) is a task in and of itself and involves coaxing the correct generations out of a LLM.
```

Comment:
```suggestion
Zero Shot Classification is a subset of [transfer learning](https://www.youtube.com/watch?v=BqqfQnyjmgg) and an attribute of Large Language Models, effectively we use a large pre-trained language model to generate a sequence of text that answers an informal question in the form of a prompt. 
```
Would be nice to add this link: https://joeddav.github.io/blog/2020/05/29/ZSL.html to Useful Resources section as we're trying to give a basic definition here without redirecting the user to a more complicated one to avoid confusion 🙂
I swapped the link for transfer learning to one of our videos as it's less overwhelming compared to a wikipedia page. In here, the end goal is to give basic information to people and at useful resources we redirect them to advanced ones if they want to learn more. The audience includes someone who wants to build something with this task without dealing with internals at the first place 🙂
I found prompt engineering to be overwhelming from first paragraph as well. Can you also divide long sentences into shorter ones (it's a technical writing rule of thumb 🙂)

You can check this page out for inspiration for the language we want to use: https://github.com/huggingface/hub-docs/blob/main/tasks/src/zero-shot-image-classification/about.md